engine/src/main/java/com/datatorrent/stram/StramUtils.java

```diff
@@ -133,4 +135,37 @@ public static JSONObject getStackTrace()
     return jsonObject;
   }
 
+  public static JSONObject getStackTrace2() throws JSONException
+  {
+
+    String result = "hello";
+    try {
+
+      String processName = java.lang.management.ManagementFactory.getRuntimeMXBean().getName();
+      Long processId = Long.parseLong(processName.split("@")[0]);
+
+      String cmd = "jstack " + processId;
```

Comment:
jstack needs to in the path, else above command might fail. also need to make sure that these tool works with different implementation of jvm.